package.json

```diff
@@ -13,6 +13,7 @@
     "lint": "tslint -p . -c tslint.json -t verbose",
     "lint-autofix": "tslint -p . -c tslint.json -t verbose --fix",
     "generate:api-definitions": "rimraf definitions/backend && mkdir -p definitions/backend && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v0.0.22/api_proxy.yaml --out-dir ./definitions/backend",
+    "generate:pagopa-api-definitions": "rimraf definitions/pagopa && mkdir -p definitions/pagopa && gen-api-models --api-spec http://pagopa-mock-rest.eu-gb.mybluemix.net/api-docs --out-dir ./definitions/pagopa --strict false",
```

Comment:
I suggest to generate the definitions from the real REST API, [here](https://acardste.vaservices.eu:1443/pp-restapi/v2/api-docs), since it's already available

Reply:
This used to point there: then, last Thursday/Friday that page was returning a 503 and the definitions were no longer available, so I updated to this. I will update this back to the original source